internal/model/model.go

```diff
@@ -1138,7 +1139,7 @@ func (m *Model) AddFolder(cfg config.FolderConfiguration) {
 		m.deviceFolders[device.DeviceID] = append(m.deviceFolders[device.DeviceID], cfg.ID)
 	}
 
-	ignores := ignore.New(m.cfg.Options().CacheIgnoredFiles)
```

Comment:
Options() locks the config wrapper, but the config wrapper is already locked when this is called from within the config commit routine... So we need to stash that variable somewhere else rather than grabbing it here.